docs/product/performance/mobile-vitals/app-starts.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "App Starts"
+sidebar_order: 50
+description: "Learn more about App Starts and how to use it to identify slow or regressed screens."
+---
+
+The **App Starts** page shows an overview of how long it takes for your application to complete cold and warm starts. Here you can identify slow or regressed screens and get additional information to understand the factors contributing to the slowness of your application start times.
+
+### Minimum SDK Requirements:
+
+**For Android:**
+
+TODO: Fill in Android SDK requirements here
```

Comment:
Minimum version: 7.4.0

And `perf-v2` has to be enabled as well for getting all new spans:

```kotlin
SentryAndroid.init(context) {
    it.dsn = "..."
    it.isEnablePerformanceV2 = true
}
```

Reply:
We can also link the SDK docs for more details - once they are written 😅

Reply:
iOS has performaceV2 since `8.18.0`, see https://docs.sentry.io/platforms/apple/performance/instrumentation/automatic-instrumentation/. App start profiling will be available from 8.21.0.